packages/better-auth/src/plugins/anonymous/index.ts

```diff
@@ -84,7 +85,10 @@ export const anonymous = (options?: AnonymousOptions | undefined) => {
 					}
 
 					const email = await getAnonUserEmail(options);
-					const name = (await options?.generateName?.(ctx)) || "Anonymous";
+					const name =
```

Comment:
This also seems unnecessary. If you really want to check if generateName is a function, please put it into the init step